### PR TITLE
remove aws/vpc-public-only module (for 8.0.0)

### PR DESCRIPTION
**Pull request by briskt**
### Removed
- Removed `aws/vpc-public-only` module. It is no longer in active use, from what I could find.

### CODE REVIEW

**Comment by Baggerone** [comment]
I'll leave it to @dalenewby to approve this one too, if that's OK.

**Review by dalenewby — APPROVED** [review]
I couldn't find any instances where I currently use this either.

**Pull request merged**
